tensorboard/webapp/metrics/views/card_renderer/image_card_component.ng.html

```diff
@@ -91,10 +91,7 @@
       [value]="stepIndex"
       (input)="onSliderInput($event)"
     ></mat-slider>
-    <div
-      class="linked-time-wrapper"
-      *ngIf="linkedTimeSelection && !linkedTimeSelection.clipped"
-    >
+    <div class="linked-time-wrapper" *ngIf="linkedTimeSelection">
```

Comment:
Do we need to check if endStep === null here?

Reply:
Are you suggesting removing the null check on line 98 -> 95?

The behavior is the same with and without it.

![50b2833a-7856-4ea6-bffb-562c25393ca6](https://user-images.githubusercontent.com/78179109/193696306-ccd10da4-47e5-44f9-9c15-bdbfb28a10c3.gif)